web/json_options.go

```diff
@@ -75,6 +75,7 @@ func defaultJSONOptions() *jsonOptions {
 			{"01-02-2006", false},
 			{"01/02/2006", false},
 			{"01/02/06", false},
+			{"SGNLUnixSec", false}, // Unix timestamp representing seconds since 1970-01-01 00:00:00 UTC.
```

Comment:
Save the name into a `const`. This would allow the adapter developers to add the JSON option like
```
web.WithDateTimeFormats(
			[]web.DateTimeFormatWithTimeZone{
				{Format: web.{constName}, HasTimeZone: false},
			}...,
		),
```